index.bs

```diff
@@ -122,8 +127,8 @@ handle additional, application-specific use cases.
     SecureContext
   ] interface Sanitizer {
     constructor(optional SanitizerConfig config = {});
-    DOMString sanitizeToString(DOMString input);
-    DocumentFragment sanitize(DOMString input);
+    DocumentFragment sanitize(SanitizerInput input);
```

Comment:
SanitizerInputWithTrustedHTML?

Reply:
I suspect not. I'm trying to handle Trusted Types integration in #49, which indeed changes this method to require TrustedHTML.

The problem I'm having is that TT is not presently an accepted standard and I think Sanitzer needs to be usable without TT. WIth 'usable' I mean not only callable from JS without TT, but also specified without relying on the TT spec draft. That is a bit awkward, honestly, but I'm not seeing any good way around this.